display-guild.lic

```diff
@@ -0,0 +1,86 @@
+Contacts = YAML.load_file('./scripts/data/base-contacts.yaml')

```

Comment:
Our convention has been to have a documentation block at the top of each script, something like:
```
=begin
  Documentation: https://elanthipedia.play.net/Lich_script_repository#accept-sell
=end
```

And a corresponding entry on elanthipedia.